frontend/README.md

```diff
@@ -14,14 +15,36 @@ This directory uses [Turborepo](https://turbo.build/) to manage the monorepo and
 
 ### Apps
 
-- `apps/design-system-storybook`: A [Storybook](https://storybook.js.org/) instance for the Code.org design system
+Open source Code.org applications:
+
+- [@code-dot-org/marketing](apps/marketing): Code.org's marketing site (uses Contentful).
+- [@code-dot-org/design-system-storybook](apps/design-system-storybook): A [Storybook](https://storybook.js.org/)
+  instance for the Code.org design system (`@code-dot-org/component-library`). Publicly available at
+  [https://code-dot-org.github.io/code-dot-org/component-library-storybook](https://code-dot-org.github.io/code-dot-org/component-library-storybook).
 
 ### Packages
 
-- `packages/component-library`: The Design System React component library for Code.org
-- `packages/component-library-styles`: Styles and theming based on the Code.org Design System
+Publicly available packages:
+
+- [@code-dot-org/component-library](packages/component-library): The Code.org Design System React component library.
+- [@code-dot-org/component-library-styles](packages/component-library-styles): Common Styles
+  (`variables`, `colors`, `mixins`, `typography styles`, etc) of Code.org Design System
+  ([@code-dot-org/component-library](packages/component-library)). Based on [Figma](https://www.figma.com/design/NIVcvUgU3WmXpAmp9U2vVy/DSCO-Variables?node-id=2925-33951&m=dev).
+  Used by [@code-dot-org/component-library](packages/component-library), can(for Code.org - should) also be used as
```

Comment:
The parenthetical is confusing and I think the first part implies it should be used for Code.org.
```suggestion
  Used by [@code-dot-org/component-library](packages/component-library), can also be used as
```